Parse-Dashboard/parse-dashboard-config.json

```
@@ -7,7 +7,18 @@
       "appName": "",
       "iconName": "",
       "primaryBackgroundColor": "",
-      "secondaryBackgroundColor": ""
+      "secondaryBackgroundColor": "",
+      "deleteOptions": {
+        "class": true,
+        "columns": true,
+        "selectedRows": true,
+        "allData": true
+      },
+      "exportOptions": {
+        "schema": true,
+        "selectedRows": true,
+        "allData": true
+      }
```

The names should be less ambiguous and follow the syntax of existing options:

```suggestion
      "deletePreference": {
        "enableDeleteClass": true,
        "enableDeleteColumns": true,
        "enableDeleteSelectedRows": true,
        "enableDeleteAllRows": true,
      },
      "exportPreference": {
        "enableExportSchema": true,
        "enableExportSelectedRows": true,
        "enableExportAllRows": true,
      }
```